CMakeLists.txt

```diff
@@ -1,33 +1,38 @@
 project(cmakesample CXX)
-cmake_minimum_required(VERSION 2.8.11)
+cmake_minimum_required(VERSION 3.5)
 
-find_package(Qt5Qml REQUIRED)
+find_package (Qt5 COMPONENTS Core Network Qml Gui Quick REQUIRED)
 
 include(FindPkgConfig)
 pkg_search_module(SAILFISH sailfishapp REQUIRED)
+add_definitions(-DSAILFISHAPP)
```

Comment:
This is not used yet, better not introduce before that.

Reply:
done